
std_msgs/Float64MultiArray velocities
std_msgs/Float64MultiArray phis
